modules/dxp/apps/saml/saml-web/src/main/java/com/liferay/saml/web/internal/struts/BaseSamlStrutsAction.java

```diff
@@ -88,23 +88,18 @@ public String execute(
 		return null;
 	}
 
-	public boolean isEnabled() {
-		return samlProviderConfigurationHelper.isEnabled();
-	}
-
-	public void setSamlProviderConfigurationHelper(
-		SamlProviderConfigurationHelper samlProviderConfigurationHelper) {
+	public abstract SamlProviderConfigurationHelper
```

Comment:
@W2jx Why this is a public method? This is an internal class, no one is going to use this public method.